utils/fabric/get-fabric-container-name.go

```diff
@@ -0,0 +1,27 @@
+// Copyright Intel Corp. 2019 All Rights Reserved.
```

Comment:
ehm picky comment: let's try to be consistent with #95 license header format for go files 💃

Reply:
i thought we want to have short header? In any case, license check passes?

Reply:
btw: shouldn't the *.md files also have headers?

Reply:
<!---
Licensed under Creative Commons Attribution 4.0 International License
https://creativecommons.org/licenses/by/4.0/
--->

Reply:
will add ...

Reply:
argh ... 
I suggest that we do follow the Hyperledger approach and add this to the project main readme

```
Hyperledger Project documentation files are made available under the Creative Commons Attribution 4.0 International License (CC-BY-4.0), available at http://creativecommons.org/licenses/by/4.0/.
```
see https://github.com/hyperledger/fabric

That way I dont see need to put license headers in *.md files.